geode-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -188,6 +188,13 @@ private void executeCommand(ChannelHandlerContext ctx, Command command) throws E
       return;
     }
 
+    if (command.isOfType(RedisCommandType.SELECT)
```

Comment:
I don't understand why this code was added. These commands are on our unimplemented list. Was this added to this PR by accident?

Reply:
Jens said that code was mistakenly added. It has now been removed